tests/geo/test_curvefilament.py

```diff
@@ -0,0 +1,116 @@
+import unittest
+from simsopt.util.zoo import get_ncsx_data
+from simsopt.geo.curvefilament import CurveFilament, FilamentRotation, create_multifilament_grid
+import numpy as np
+
+
+class MultifilamentTesting(unittest.TestCase):
+
```

Comment:
Would it be worth adding a test case in this file which is basically the Taylor test in `stage_two_optimization_finitebuild.py` but with asserts? If the Taylor test in that example fails, presently no error is raised. The Taylor test in that example ensures that the finite build machinery works even when applying nfp-symmetry and stellarator-symmetry and plugging the finite-build coils into Biot-Savart, and it's not obvious that the other tests in `test_curvefilament.py` cover those aspects.